packages/react/src/oxygen-ui-react-auth-components/SignInButton/sign-in-button.scss

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.OxygenSignInButton {
+  margin: 16px 0;
+}
+
+.OxygenSignInButtonSocial {
+  background: white;
+  color: black;
+  margin: 0;
+  padding: 8px 0;
+  /* stylelint-disable-next-line color-function-notation */
```

Comment:
Shall we not suppress the stylelint rules?

Reply:
removed with b2837c37c77e5a9c0a1bf501ec5e78d5102b5dd2